src/Chainweb/RestAPI/Orphans.hs

```diff
@@ -195,3 +195,9 @@ instance FromHttpApiData RewindDepth where
 
 instance ToHttpApiData RewindDepth where
     toUrlPiece (RewindDepth k) = toUrlPiece k
+
+instance FromHttpApiData ConfirmationDepth where
+    parseUrlPiece = fmap ConfirmationDepth . parseUrlPiece
+
+instance ToHttpApiData ConfirmationDepth where
+    toUrlPiece (ConfirmationDepth k) = toUrlPiece k
```

Comment:
I didn't think of this earlier with the depth/limit PR, but how about something like:
```suggestion
deriving newtype instance FromHttpApiData ConfirmationDepth

deriving newtype instance ToHttpApiData ConfirmationDepth
```
You can replace some other instances in this file in that way if it works too.

Reply:
Yep, updated